src/LayerEditor/ui/panels/surfaces.py

```diff
@@ -619,11 +515,29 @@ def _check_file_path(self, file):
 
     def apply(self):
         """Save changes to file and compute new elevation and error"""
+        msg = QtWidgets.QMessageBox(QtWidgets.QMessageBox.Information,
```

Comment:
Good to have, such notification.

Reply:
Yes, we discussed this. Since Radek improved grid generetor it takes a lot of time to compute.